Tofu/AccountCell.swift

```diff
@@ -55,6 +55,7 @@ private func imageForAccount(_ account: Account) -> UIImage {
     case .some("Fastmail"): return UIImage(named: "FastMail")!
     case .some("Firefox"): return UIImage(named: "Firefox")!
     case .some("gandi.net"): return UIImage(named: "Gandi")!
+		case .some("Gitea"): return UIImage(named: "Gitea")!
```

Comment:
I would prefer if this line was indented with four spaces (instead of two tabs) to match the indentation of adjacent lines.